Function "secrets" not defined

After starting a new repo with this template I run `make setup`. but there is an issue with the GitHub token for the goreleaser. I'm not sure if this a requirement of my system that I'm missing, as I haven't worked with GitHub actions before. I created a Personal Access Token and exported its value as the `GITHUB_TOKEN` environment variable locally but that didn't change the error. 

This is the output from running `make setup`:
emphasis on `template: release.yml:22: function "secrets" not defined`

```
$ make setup 
make -C setup
make[1]: Entering directory '/home/brian/Projects/kubectl-viz/setup'
go run ./...
GitHub Organization or Usernamebrianrudolf
GitHub Repo Namekubectl-viz
Plugin Namekubectl-viz
Updating README
Updating sample code with names
template: release.yml:22: function "secrets" not defined
exit status 1
make[1]: *** [Makefile:6: all] Error 1
make[1]: Leaving directory '/home/brian/Projects/kubectl-viz/setup'
make: *** [Makefile:29: setup] Error 2
```

I think this is more of a GitHub secrets error I'm running into but I'm hoping someone here can point me in the right direction and we can edit the README to add this as a prerequisite for this template.

--- a/setup/templates.go
+++ b/setup/templates.go
@@ -34,6 +34,13 @@ func renderTemplates(templateContext TemplateContext) error {
 				}
 			}
 
+			// Ignore .github
+			if len(pathSplit) >= 2 {
+				if pathSplit[1] == ".github" {
+					return nil
+				}
+			}
+
 			fi, err := os.Stat(path)
 			if err != nil {
 				return errors.Wrap(err, "failed to read file info")

